src/legacy/core_plugins/kibana/server/tutorials/coredns_logs/index.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n }  from '@kbn/i18n';
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { onPremInstructions, cloudInstructions, onPremCloudInstructions } from '../../../common/tutorials/filebeat_instructions';
+
+export function corednsLogsSpecProvider(server, context) {
+  const moduleName = 'coredns';
+  const platforms = ['OSX', 'DEB', 'RPM'];
+  return {
+    id: 'corednsLogs',
+    name: i18n.translate('kbn.server.tutorials.corednsLogs.nameTitle', {
+      defaultMessage: 'Coredns logs',
```

Comment:
Consider formatting `Coredns` as `CoreDNS` ( per https://coredns.io/manual/toc/ ) for the human-readable text

Reply:
Sure, I will update it. thanks